openverse-api/catalog/api/serializers/image_serializers.py

```diff
@@ -110,6 +107,12 @@ class ImageSerializer(MediaSerializer):
     )
 
     # Hyperlinks
+    thumbnail = serializers.HyperlinkedIdentityField(
```

Comment:
How is the actual url path generated for these hyperlinks?

Reply:
It reverses the value of the `view_name` kwarg, and then populates the value of the `lookup_field` kwarg in the path params.